ui/src/utils/fetchWrappers/systemInfoFetch.ts

```diff
@@ -23,12 +20,17 @@ export const useSystemInfoFetch = () => {
       setLoading(false);
       return;
     }
+    if (data?.errMsg) {
```

Comment:
In the other fetchwrappers I just combine error and errMsg, into just error in the result. Can we do that here too? The components using it will need to do less "error" checks this way.